ocaml/lib/pickles_bindings.ml

```diff
@@ -602,12 +744,37 @@ let dummy_verification_key () =
   let hash = Mina_base.Zkapp_account.digest_vk vk in
   (data |> Js.string, hash)
 
+let encode_verification_key (vk : Pickles.Verification_key.t) =
+  Pickles.Verification_key.to_yojson vk |> Yojson.Safe.to_string |> Js.string
+
+let decode_verification_key (bytes : Js.js_string Js.t) =
+  let vk_or_error =
+    Pickles.Verification_key.of_yojson @@ Yojson.Safe.from_string
+    @@ Js.to_string bytes
+  in
+  let open Ppx_deriving_yojson_runtime.Result in
+  match vk_or_error with
+  | Ok vk ->
+      vk
+  | Error err ->
+      failwithf "Could not decode verification key: %s" err ()
+
+module Util = struct
```

Comment:
It seems to be a simple alias. Is it really useful?

Reply:
I could put it in the `js%object` declaration, that's true. currently there is a consistent style of not putting any logic in js object declaration, and _only_ using aliases there. in this case I could make an exception